components/home-two/home-two-tags-selector.tsx

```
@@ -0,0 +1,219 @@
+            <span className="text-sm font-bold">
+              {currentTag ? currentTag.name : t("listing.TAGS")}
+            </span>
+            {currentTag && (
+              <>
+                <span className="text-xs text-gray-500 dark:text-gray-400">
+                  ({currentTag.count})
+                </span>
+                <span
+                  onClick={handleClearTag}
+                  className="ml-1 p-1 rounded-full hover:bg-gray-200 dark:hover:bg-gray-700 transition-colors cursor-pointer"
+                  role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)